cmake/Modules/G4DeveloperAPI.cmake

```diff
@@ -933,7 +933,7 @@ function(geant4_compose_targets)
     geant4_get_module_property(__parent_target ${__module} PARENT_TARGET)
     get_filename_component(__listdir "${__listfile}" DIRECTORY)
     # Remove generated headers
-    list(TRANSFORM __publichdrs REPLACE "^${PROJECT_BINARY_DIR}/.*$" "")
+    #list(TRANSFORM __publichdrs REPLACE "^${PROJECT_BINARY_DIR}/.*$" "")
```

Comment:
What's the value of `PROJECT_BINARY_DIR` here? The regex is very simple in that is should just remove the value of that variable at the beginning of the input string.

Reply:
The value of PROJECT_BINARY_DIR is D:/Documents/_C++_projects/geant4-v11.2.1-build.
CMake fails on this line with a:
```
[INFO] RegularExpression::compile(): Nested *?+.
[INFO] RegularExpression::compile(): Error in compile.
```